doc/readme/ABOUT.md

```diff
@@ -2,16 +2,16 @@ Shogun is an open-source machine learning library that offers a wide range of ef
 
 #### Shogun is accessible
 
-* Supports many languages (Python, Octave, R, Java/Scala, Lua, C#, Ruby, etc) and platforms (Linux/Unix, MacOS and Windows) and integrates with their scientific computing environments.
+* Supports many languages (Python, Octave, R, Java/Scala, Lua, C#, Ruby, etc) and platforms (Linux/Unix, macOS, and Windows) and integrates with their scientific computing environments.
 
 #### Shogun is state-of-the-art
 
 * Efficient implementation (from standard to cutting edge algorithms), modern software architecture in C++.
-* Easy combination of multiple data representations, algorithm classes and general purpose tools for rapid prototyping of data pipelines.
+* Easy combination of multiple data representations, algorithm classes, and general-purpose tools for rapid prototyping of data pipelines.
```

Comment:
I guess the Oxford comma is a matter of opinion :D

Reply:
I agree with you but there are some pros to do this: the first is GitHub's use of Oxford comma (even in the frequently observed message _"Learn Git and GitHub without any code!
Using the Hello World guide, you’ll start a branch, write comments, and open a pull request."_, the second is having some number of sentences with the comma in docs e.g. https://github.com/shogun-toolbox/shogun/pull/4906/files#diff-f5f6b994e6c7890608d6a4dfacbef362L105

Reply:
true! I don't have an opinion on it tbh, just an observation :)